src/styles/styles.scss

```diff
@@ -23,7 +23,10 @@
 }
 
 // NotificationSystem component
-.notifications-system {
+:local .notifications-system {
```

Comment:
Hmm... I'm not sure it's a good choice. Wybo theme could have some different font styles than bootstrap. Correct me if i'm wrong :)